.github/settings.yml

```
@@ -123,15 +123,15 @@ branches:
       # Required. Require status checks to pass before merging. Set to null to disable
       required_status_checks:
         # Required. Require branches to be up to date before merging.
-        strict: true
+        strict: false
         # Required. The list of status checks to require in order to merge into this branch
-        contexts: [ "bors" ]
+        contexts: []
       # Required. Enforce all configured restrictions for administrators. Set to true to enforce required status checks for repository administrators. Set to null to disable.
       enforce_admins: false
       # Disabled for bors to work
       required_linear_history: false
       # Required. Restrict who can push to this branch. Team and user restrictions are only available for organization-owned repositories. Set to null to disable.
       restrictions:
-        apps: [ "bors" ]
+        apps: []
```

I think mergify needs permissions like bors.

```suggestion
        apps: [ "mergify" ]
```

---

This means that only mergify can push to main. Is that what we want?